README.md

```diff
@@ -4,23 +4,19 @@ Status: **experimental**
 
 This is an [OCurrent][] pipeline that tests submissions to [opam-repository][].
 
-To test locally you will need:
-
-1. A [personal access token][] from GitHub.
-2. A `submission.cap` for an [OCluster][] build cluster.
-
-Run the `opam-repo-ci-local` command
-(you might need to increase the limit on the number of open files):
+To test locally you will need a local copy of the [opam-repository][] Git repo. Run the `opam-repo-ci-local` command (you might need to increase the limit on the number of open files):
 
 ```
 ulimit -n 102400
 dune exec -- opam-repo-ci-local \
   --confirm harmless \
-  --submission-service submission.cap \
-  --github-token-file token \
+  --repo REPO-PATH \
+  --branch BRANCH-NAME \
   --capnp-address tcp:127.0.0.1:5001
```

Comment:
```suggestion
```

This parameter isn't needed.

Reply:
Just to clarify, it isn't needed for the frontend web interface?

Reply:
This is what I get
```
$ dune exec -- opam-repo-ci-local --confirm harmless --repo ../opam-repository/ --branch test --capnp-address tcp:127.0.0.1:5001
opam-repo-ci-local: unknown option '--capnp-address'.
Usage: opam-repo-ci-local [OPTION]…
Try 'opam-repo-ci-local --help' for more information.
```

But without the parameter it works fine.